packages/netlify-cms-core/src/actions/config.js

```diff
@@ -111,12 +106,15 @@ export function loadConfig() {
     try {
       const preloadedConfig = getState().config;
       const configUrl = getConfigUrl();
-      const loadedConfig = await getConfig(configUrl, preloadedConfig && preloadedConfig.size > 1);
+      const loadedConfig =
+        bootstrapConfig && bootstrapConfig.ignoreLoadConfig
```

Comment:
Snake case is used for our yaml config keys, and we try to phrase keys in the positive, so this should probably be `load_config_file`, defaulting to `true`, and we check for a `false` value to skip loading.